.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup NodeJS
+        uses: actions/setup-node@v2
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install solhint
+        run: npm install -g solhint
+
+      - name: Run linter
+        run: solhint 'contracts/Staking.sol'
```

Comment:
Can you define this script in package.json so that we can call `npm run lint` here